internal/build/buildconfig/maker.go

```diff
@@ -51,11 +52,22 @@ func (m *maker) MakeBuildConfig(mod kmmv1beta1.Module, mapping kmmv1beta1.Kernel
 		buildTarget = buildv1.BuildOutput{}
 	}
 
+	sourceConfig := buildv1.BuildSource{
+		Dockerfile: &kmmBuild.Dockerfile,
+		Type:       buildv1.BuildSourceDockerfile,
+	}
```

Comment:
So it means that if the user has changed some data in the BuildConfig which is not the Dockerfile itself, the build won't be retrigerred?!

Don't we want to hash the full buildConfig?

Reply:
The data that user can change ( in the Module spec) is only used in the BuildSource. All the rest are set by operator code. So, in case of Module change, we only need to compare hashes of the BuildSource

Reply:
Makes perfect sense. Thanks.